.jshintrc

```diff
@@ -0,0 +1,74 @@
+{
+    "bitwise"       : true,
+    "curly"         : true,
+    "eqeqeq"        : true,
+    "forin"         : true,
+    "immed"         : true,
+    "newcap"        : true,
+    "noarg"         : true,
+    "noempty"       : true,
+    "regexp"        : true,
+    "undef"         : true,
+    "strict"        : true,
+    "validthis"     : true,
+
+    "asi"           : false,
+    "boss"          : false,
+    "debug"         : false,
+    "eqnull"        : false,
+    "es5"           : false,
+    "esnext"        : false,
+    "evil"          : false,
+    "expr"          : false,
+    "funcscope"     : false,
+    "globalstrict"  : false,
+    "iterator"      : false,
+    "lastsemic"     : false,
+    "latedef"       : false,
+    "laxbreak"      : false,
+    "laxcomma"      : false,
+    "loopfunc"      : false,
+    "multistr"      : false,
+    "nonew"         : false,
+    "onecase"       : false,
+    "plusplus"      : false,
+    "proto"         : false,
+    "regexdash"     : false,
+    "scripturl"     : false,
+    "shadow"        : false,
+    "sub"           : false,
+    "supernew"      : false,
+    "unused"        : false,
+
+    "browser"       : false,
+    "couch"         : false,
+    "devel"         : true,
+    "dojo"          : false,
+    "jquery"        : false,
+    "mootools"      : false,
+    "node"          : false,
+    "nonstandard"   : false,
+    "prototypejs"   : false,
+    "rhino"         : false,
+    "wsh"           : false,
+
+    "maxerr"        : 100,
+    "indent"        : 4,
+    "globals"       : {
+        "window": false,
+        "document": false,
+        "setTimeout": false,
+        "require": false,
+        "define": false,
+        "brackets": false,
+        "$": false,
+        "PathUtils": false,
+        "window": false,
+        "navigator": false,
+        "Mustache": false,
+        "Bramble": false,
+        "ArrayBuffer": false,
+        "module": false,
+        "Uint8Array": false
```

Comment:
This is almost like a global declaration of which global variables jshint should ignore.